catalog-info.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: davidia
+  title: davidia
+  description: React components for a data visualiser
+  annotations:
+    diamond.ac.uk/viewdocs-url: https://diamondlightsource.github.io/davidia/
+    github.com/project-slug: DiamondLightSource/davidia
+spec:
+  type: library
+  lifecycle: production
+  owner: user:vaq49247
```

Comment:
Have to see if we can set this to something more public...

Reply:
Some have the owner set to a group instead.

Reply:
Ok, use `group:data-analysis`